Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -126,15 +126,14 @@ void IBocColumnRenderer.RenderTitleCell (BocColumnRenderingContext renderingCont
         if (!string.IsNullOrEmpty (columnItemID))
           renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.ItemID, columnItemID);
 
-        var columnTitle = renderingContext.ColumnDefinition.ColumnTitleDisplayValue;
-        if (!string.IsNullOrEmpty (columnTitle))
-          renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));
+        var columnTitle = StringUtility.NullToEmpty (renderingContext.ColumnDefinition.ColumnTitleDisplayValue);
+        renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));
 
         var oneBasedCellIndex = renderingContext.VisibleColumnIndex + 1;
         renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributesForObjectBinding.BocListCellIndex, oneBasedCellIndex.ToString());
 
         renderingContext.Writer.AddAttribute (
-            DiagnosticMetadataAttributesForObjectBinding.BocListColumnHasDiagnosticMetadata,
+            DiagnosticMetadataAttributesForObjectBinding.BocListColumnHasContentAttribute,
             HasDiagnoticMetadata.ToString().ToLower());

```

Comment:
Rename.

Reply:
We have one more problem: in SimpleValueColumnRendererer, the content attribute is not rendered if the column is in edit mode. Am not sure how to best solve this. Maybe we just need to render the current value as attribute in SimpleValueColumnRenderer.RenderEditModeControl().

Reply:
@MichaelKetting  Would this be the desired outcome?

- Edit mode:
![image](https://user-images.githubusercontent.com/40357550/72147426-c61f2280-339e-11ea-920b-aaa3fd0027f0.png)

- Normal mode for reference:
![image](https://user-images.githubusercontent.com/40357550/72147489-ea7aff00-339e-11ea-9bbe-4066e63ff273.png)

Reply:
For the sake of completeness, that's what the nesting looks like with commands.

- Display name with command:
![image](https://user-images.githubusercontent.com/40357550/72151276-18b10c80-33a8-11ea-8ada-2378fb9ec5c9.png)

- Command (no `data-boclist-cell-contents` - makes sense):
![image](https://user-images.githubusercontent.com/40357550/72151354-4a29d800-33a8-11ea-9c10-deea2df23553.png)